### ISSUE TITLE
glance_image doesn't support "disk_format=ami"

### ISSUE THREAD

**Issue by yfried** [ansible/ansible-modules-core#708]
glance_image limits the input for disk_format. should be delegated to glance API

failed: [...] => {"failed": true}
msg: value of disk_format must be one of: aki,vhd,vmdk,raw,qcow2,vdi,iso, got: ami

**Comment by ansibot** [comment]
@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by rcarrillocruz** [comment]
Hi @yfried-redhat 

glance_image module is considered deprecated.

Can you please use os_image instead? The disk_format is exposes as a parameter.

Thanks

**Comment by Shrews** [comment]
As @rcarrillocruz said, this module is deprecated.

wontfix
